### PR TITLE
[libhv] update to v1.2.1

### PR DESCRIPTION
**Pull request by ithewei**
**Describe the pull request**

- #### What does your PR fix?  
  The PR updates libhv library to v1.2.1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  `all`, `Yes`

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  `Yes`

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
@ithewei ,Could you plesae take a look?

CMake Error at ports/libhv/portfile.cmake:32 (file):
  file RENAME failed to rename

    D:/packages/libhv_x86-windows/lib/hv.dll

  to

    D:/packages/libhv_x86-windows/bin/hv.dll

  because: The system cannot find the path specified.

File: ports/libhv/portfile.cmake [hunk 1]
```diff
@@ -27,3 +27,17 @@ vcpkg_install_cmake()
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+
+if(EXISTS "${CURRENT_PACKAGES_DIR}/lib/hv.dll")
+    if(NOT EXISTS "${CURRENT_PACKAGES_DIR}/bin")
+        file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/bin")
```
Comment (JonLiu1993) [inline ports/libhv/portfile.cmake:33]: @ithewei ,I saw that you created "${CURRENT_PACKAGES_DIR}/bin" directly in the portfile.cmake file when you fixed the error “when the specified path could not be found”. There may be some problems with this. Could you please use patch to generate it in CMakeLists.txt using install Does the bin directory link let it exist in it?

**Review by JonLiu1993 — APPROVED** [review]

File: ports/libhv/portfile.cmake [hunk 2]
```diff
@@ -27,3 +27,17 @@ vcpkg_install_cmake()
...
+    endif()
```
Comment (BillyONeal) [inline ports/libhv/portfile.cmake:34]:
```suggestion
    file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/bin")
```

File: ports/libhv/portfile.cmake [hunk 3]
```diff
@@ -27,3 +27,17 @@ vcpkg_install_cmake()
...
+    file(RENAME "${CURRENT_PACKAGES_DIR}/lib/hv.dll" "${CURRENT_PACKAGES_DIR}/bin/hv.dll")
+endif()
+
+if(EXISTS "${CURRENT_PACKAGES_DIR}/debug/lib/hv.dll")
+    if(NOT EXISTS "${CURRENT_PACKAGES_DIR}/debug/bin")
+        file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/bin")
+    endif()
```
Comment (BillyONeal) [inline ports/libhv/portfile.cmake:41]:
```suggestion
    file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/debug/bin")
```

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**